tests/testthat/test_surv_ranger.R

```diff
@@ -1,33 +1,36 @@
 context("surv_ranger")
 
 ## TODO: Proper test required when predictions working
+## Suppress 'experimental' warning for splirule = maxstat 
```

Comment:
IMHO we should leave out the test for splitrule = "maxstat" as this is experimental.